README.md

```diff
@@ -1,3 +1,78 @@
 # Ansible role: Helm
 
 This role handles the creation and updates of Helm charts.
+
+[![Build Status](https://travis-ci.com/Flaconi/ansible-role-helm.svg?branch=master)](https://travis-ci.com/Flaconi/ansible-role-helm)
+[![Version](https://img.shields.io/github/tag/Flaconi/ansible-role-helm.svg)](https://github.com/Flaconi/ansible-role-helm/tags)
+[![Ansible Galaxy](https://img.shields.io/ansible/role/d/25936.svg)](https://galaxy.ansible.com/Flaconi/aws-vpc-cgw/)
+
+## Requirements
+
+* Ansible 2.5
+
+
+## Additional variables
+
+Additional variables that can be used (either as `host_vars`/`group_vars` or via command line args):
+
+| Variable                     | Description              |
+|------------------------------|--------------------------|
+| `helm_version`               | Helm version to download |
+| `helm_local_directory_path`  | Directory into which to download Helm |
+| `helm_configuration_files`   | Directory where chart configuration files is stored |
```

Comment:
I would add `helm_configuration_files` and `helm_charts` as these are the values the user would want to set for this role.

Reply:
Good catch. Done!